SU-HSDO/suhumsci
# France bugherd fixups

# READY FOR REVIEW

# Summary
- Break words inside of card patterns to keep from running outside of container
- Remove margin from last HTML element in card patterns
- Enhance optical alignment of buttons by increasing bottom padding
- Update default for accent dash on given page layout; update Archaeology and France Stanford default colors in custom stylesheets
- Remove card styles from person horizontal card on custom Project content type
- Add spacing between person card and main content for custom Project content type
- Add spacing beneath masthead/header and add offset for hero paragraph item when it is the first one used
- Add space between inline labels

# Needed By (Date)
- End of sprint

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Test to ensure that above noted changes are reflected; I can provide examples to corresponding items as needed

# Affected Projects or Products
- suhumsci (theme)
- France-Stanford

# Associated Issues and/or People
- PR #111 should be merged before this, as this work builds on that
- https://stanfordits.atlassian.net/browse/HSD8-397
- https://stanfordits.atlassian.net/browse/HSD8-400
- https://stanfordits.atlassian.net/browse/HSD8-402
- https://stanfordits.atlassian.net/browse/HSD8-403
- https://stanfordits.atlassian.net/browse/HSD8-404
- https://stanfordits.atlassian.net/browse/HSD8-405
- https://stanfordits.atlassian.net/browse/HSD8-406
- https://stanfordits.atlassian.net/browse/HSD8-407
- https://stanfordits.atlassian.net/browse/HSD8-408
- https://stanfordits.atlassian.net/browse/HSD8-410
- https://stanfordits.atlassian.net/browse/HSD8-413
- https://stanfordits.atlassian.net/browse/HSD8-414

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
